tests/test_views.py

```diff
@@ -1013,6 +1015,20 @@ def _setup_enrollment_client(self, client_mock):
         client.get_course_modes.return_value = self.dummy_demo_course_modes
         client.get_course_enrollment.return_value = None
 
+    def _setup_ecommerce_client(self, client_mock):
```

Comment:
@replaceafill I'm not sure why you removed the `total` argument in 783e2b9b29f1e49608f71d1b789a625a202003d9? Please add it back in and get rid of the duplication between this method and `test_get_course_enrollment_page` below.